pydantic/experimental/pipeline.py

```diff
@@ -0,0 +1,580 @@
+"""Experimental functionality, be prepared for this to break."""
+
+from __future__ import annotations
+
+import datetime
+import operator
+import re
+from collections import deque
+from collections.abc import Container
+from dataclasses import dataclass
+from decimal import Decimal
+from functools import cached_property, partial
+from typing import TYPE_CHECKING, Any, Callable, Generic, Pattern, Protocol, TypeVar, Union, overload
+
+import annotated_types
+from typing_extensions import Annotated
+
+if TYPE_CHECKING:
+    from pydantic_core import core_schema as cs
+
+    from pydantic import GetCoreSchemaHandler
+
+from pydantic._internal._internal_dataclass import slots_true
```

Comment:
```suggestion
from pydantic._internal._internal_dataclass import slots_true as _slots_true
```

And we should also update the test that checks nothing from `_internal` is available to import publicly.

Reply:
Done, I updated this function:

https://github.com/pydantic/pydantic/blob/137d4d83936bf5077c4b6354c43314bca704c4f6/tests/test_exports.py#L37-L61